helm/hpcc/templates/_warnings.tpl

```diff
@@ -0,0 +1,57 @@
+{{/*
+
+---  DO NOT EDIT THIS FILE ---
```

Comment:
Why not?!

Reply:
Not sure but it's copied from the _helpers.tpl.